src/routeConfiguration.js

```diff
@@ -118,6 +126,31 @@ const routeConfiguration = () => {
       component: EditListingPage,
       loadData: pageDataLoadingAPI.EditListingPage.loadData,
     },
+    {
+      path: '/l/newTeacher',
+      name: 'NewTeacherListingPage',
+      auth: true,
+      component: () => (
+        <NamedRedirect
+          name="EditTeacherListingPage"
+          params={{ slug: draftSlug, id: draftId, type: 'new', tab: 'general' }}
+        />
+      ),
+    },
+    {
+      path: '/l/teacher/:slug/:id/:type/:tab',
+      name: 'EditTeacherListingPage',
+      auth: true,
+      component: EditTeacherListingPage,
+      loadData: pageDataLoadingAPI.EditTeacherListingPage.loadData,
+    },
+    {
+      path: '/l/teacher/:slug/:id/:type/:tab/:returnURLType',
+      name: 'EditTeacherListingStripeOnboardingPage',
+      auth: true,
+      component: EditTeacherListingPage,
+      loadData: pageDataLoadingAPI.EditTeacherListingPage.loadData,
+    },
```

Comment:
Yes, because the logic of teacher listing is almost different from sauna so we need to separate all pages of the teacher with the sauna for more easily maintain if we want to update some feature on both pages.

Reply:
That would not be needed. Since the template is provided for us to modify to suite with the client's marketplace idea. It would make sense to separate the component if the marketplace have both sauna and teacher listing. But it does not make sense when this is a teaching platform and there should be no sauna listing in a teaching platform.

Reply:
But the requirement is to add a new teacher listing but keep using the sauna listing, the test is described that.
![image](https://user-images.githubusercontent.com/83624927/119315273-e142b780-bc9f-11eb-9591-9cde1ad0a73b.png)